packages/protocol/contracts/identity/Random.sol

```diff
@@ -13,7 +13,18 @@ contract Random is IRandom {
 
   bytes32 public _random;
 
+  uint256 public randomnessBlockRetentionWindow = 256;
+
+  mapping (uint256 => bytes32) private history;
+  uint256 private historyFirst;
+  uint256 private historySize;
+
   function initialize() external {
+    randomnessBlockRetentionWindow = 256;
+  }
+
+  function setRandomnessBlockRetentionWindow(uint value) external {
```

Comment:
`onlyOwner`, please add tests to confirm that only the owner can configure this, and turn ownership over to governance in the migrations